source/runner/Executor.ts

```diff
@@ -1,23 +1,16 @@
-// @flow
-
 import { contextForDanger } from "../runner/Dangerfile"
 import { DangerDSL } from "../dsl/DangerDSL"
-import type { CISource } from "../ci_source/ci_source"
+import { CISource } from "../ci_source/ci_source"
 import { Platform } from "../platforms/platform"
-import type { DangerResults } from "../dsl/DangerResults"
-import githubResultsTemplate from "./templates/github-issue-template"
+import { DangerResults } from "../dsl/DangerResults"
+import { template as githubResultsTemplate } from "./templates/github-issue-template"
 import { createDangerfileRuntimeEnvironment, runDangerfileEnvironment } from "./DangerfileRunner"
-import type { DangerfileRuntimeEnv } from "./types"
+import { DangerfileRuntimeEnv } from "./types"
 
 // This is still badly named, maybe it really should just be runner?
 
-export default class Executor {
```

Comment:
So, it's bit complicated issue. TS compiler does not interop `default` export, so if you export something like

`export default aaa` in a.ts

then you have to import like
```
import xxx from '...a'

xxx.default //have to explicitly specify 'default' namespace since TS compiler does not interop
```

while babel does. So it is typically recommended to use named export, unless it is expected to use explicit `x.default`everywhere.